chapter1/configuration.md

```diff
@@ -1,12 +1,12 @@
 ## Basic Configuration
 
 ### Launch webportal
-In `appliance_console`, we can check either `ManageIQ Server` is running or not. If not then we can start it by entering digit `14` in the advanced setting.  After that when we hit IPv4 address which is given in `appliance_console` in web browser we can see the ManageIQ's Log In page.
+In `appliance_console`, we need to check either `ManageIQ Server` is up. If not then we can start it by entering digit `14` in the advanced setting.  After that when we hit IPv4 address which is given in `appliance_console` in web browser we can see the ManageIQ's Login page.
```

Comment:
It should be like, `we need to check if ...`

Reply:
Rewrited sentence using word 'whether'.